bss-core/service.go

```diff
@@ -142,6 +142,19 @@ func (s *Service) eventLoop() {
 	for {
 		select {
 		case <-ticker.C:
+			startTime := time.Now().UnixMilli()
+			go func() {
+				for {
+					if time.Now().UnixMilli()-startTime > 2*60*1000 {
```

Comment:
Whether a variable name can be used instead 2*60*1000， otherwise it will confuse people